theme/themes/pxt/views/card.overrides

```diff
@@ -5,4 +5,77 @@
 .ui.card .ui.button.massive {
     border-bottom-left-radius: 0;
     border-bottom-right-radius: 0;
+}
+
+/* File card for the Home screen */
+.ui.card.file {
```

Comment:
File card? What files are we linking to?

Reply:
I refer to header as file cards.

Reply:
what is a `header`?

Reply:
a local project.